src/main/java/com/minecolonies/coremod/proxy/ClientProxy.java

```diff
@@ -264,8 +267,10 @@ public static void registerModels(@NotNull final ModelRegistryEvent event)
         }
 
         //Additionally we register an exclusion handler here;
-        TemplateRenderHandler.getInstance().registerExclusionHandler((b) -> b.blockState.getBlock() instanceof BlockSubstitution);
-
+        TemplateBlockAccessTransformHandler.getInstance().AddTransformHandler(
+          (b) -> b.blockState.getBlock() instanceof BlockSubstitution,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the parentheses around the "b" parameter (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1611)